src/DynamoDbServiceProvider.php

```diff
@@ -17,7 +17,7 @@ public function boot()
         DynamoDbModel::setDynamoDbClientService($this->app->make(DynamoDbClientInterface::class));
 
         $this->publishes([
-            __DIR__.'/../config/dynamodb.php' => config_path('dynamodb.php'),
+            __DIR__.'/../config/dynamodb.php' => app()->path('dynamodb.php'),
```

Comment:
doesn't `app()->path` give you the path to the `app` folder? shouldn't this be `app()->configPath('dynamodb.php')`?

Reply:
I think it is `app()->basePath('config/dynamodb.php')`